assets/lambda@edge/LambdaOriginRequestIamAuth.test.ts

```diff
@@ -0,0 +1,109 @@
+    const event: CloudFrontRequestEvent = {
+      Records: [
+        {
+          cf: {
+            config: {
+              distributionDomainName: '',
+              distributionId: '',
+              eventType: 'origin-request',
+              requestId: '',
+            },
+            request: {
+              clientIp: '1.1.1.1',
+              headers: {
+                host: [
+                  {
+                    key: 'Host',
+                    value: 'd6b8brjqfujeb.cloudfront.net',
+                  },
+                ],
+                'accept-language': [
+                  {
+                    key: 'Accept-Language',
+                    value: 'en-US,en;q=0.9',
+                  },
+                ],
+                referer: [
+                  {
+                    key: 'Referer',
+                    value: 'https://d6b8brjqfujeb.cloudfront.net/batches/2/overview',
+                  },
+                ],
+                'x-forwarded-for': [
+                  {
+                    key: 'X-Forwarded-For',
+                    value: '1.1.1.1',
+                  },
+                ],
+                'user-agent': [
+                  {
+                    key: 'User-Agent',
+                    value:
+                      'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36',
+                  },
+                ],
+                via: [
+                  {
+                    key: 'Via',
+                    value: '2.0 8bf94e29f889f8d0076c4502ae008b58.cloudfront.net (CloudFront)',
+                  },
+                ],
+                'accept-encoding': [
+                  {
+                    key: 'Accept-Encoding',
+                    value: 'br,gzip',
+                  },
+                ],
+                'sec-ch-ua': [
+                  {
+                    key: 'sec-ch-ua',
+                    value: '"Google Chrome";v="113", "Chromium";v="113", "Not-A.Brand";v="24"',
+                  },
+                ],
+              },
+              method: 'GET',
+              querystring: '',
+              uri: '/some/path',
+              origin: {
+                custom: {
+                  customHeaders: {},
+                  domainName: 'kjtbbx7u533q7p7n5font6gpci0phrng.lambda-url.us-east-1.on.aws',
+                  keepaliveTimeout: 5,
+                  path: '',
+                  port: 443,
+                  protocol: 'https',
+                  readTimeout: 30,
+                  sslProtocols: ['TLSv1.2'],
+                },
+              },
+              body: {
+                action: 'read-only',
+                data: '',
+                encoding: 'base64',
+                inputTruncated: false,
+              },
+            },
+          },
+        },
+      ],
+    };
+    await handler(event, {} as CloudFrontRequestContext, () => {});
```

Comment:
should be some expect call here

Reply:
My original thought was to only use this for debugging purposes, but I'll add an expect call.